en/docs/learn/logging-in-to-salesforce-using-the-identity-server.md

```diff
@@ -125,7 +125,7 @@ Server credentials.
 			<div class="codeContent panelContent pdl">
 			<pre class="html/xml" data-syntaxhighlighter-params="brush: html/xml; gutter: false; theme: Confluence" data-theme="Confluence" style="brush: html/xml; gutter: false; theme: Confluence">
 
-			<code>[user_store]<br>username_javascript_regex = "^[a-zA-Z0-9._-]+@[a-zA-Z0-9.-]+.[a-zA-Z]{2,4}$"</code></pre>
+			<code>[user_store]<br>username_javascript_regex = "^[a-zA-Z0-9._-]+@[a-zA-Z0-9.-]+.[a-zA-Z]{2,4}"</code></pre>
```

Comment:
We need to keep $, instead \$ to overcome rendering issues

Reply:
corrected